src/test/java/org/apidesign/polyfill/CryptoPolyfillTest.java

```diff
@@ -0,0 +1,85 @@
+package org.apidesign.polyfill;
+
+import java.util.Arrays;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import org.apidesign.polyfill.crypto.CryptoPolyfill;
+import org.graalvm.polyglot.Context;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CryptoPolyfillTest {
+
+    private Context context;
+    private ExecutorService executor;
+
+    public CryptoPolyfillTest() {
+    }
+
+    @Before
+    public void setup() throws Exception {
+        executor = Executors.newSingleThreadExecutor();
+        var crypto = new CryptoPolyfill();
+        var b = Context.newBuilder("js");
+
+        var chromePort = Integer.getInteger("inspectPort", -1);
+        if (chromePort > 0) {
+            b.option("inspect", ":" + chromePort);
+        }
+
+        context = CompletableFuture
```

Comment:
This feels like a bit too complicated initialization for end users.

Reply:
yes, it feels quite verbose